lms/djangoapps/certificates/tests/test_api.py

```diff
@@ -146,6 +150,19 @@ def test_xqueue_submit_task_error(self):
         self.assertEqual(cert.status, 'error')
         self.assertIn(self.ERROR_REASON, cert.error_reason)
 
+    @override_settings(FEATURES=FEATURES_WITH_CERTS_ENABLED)
```

Comment:
You should be able to set just a single flag rather than having to make a copy of all the features. If you grep the code, you'll see a number of example. I think it uses @mock.patch()...